workspace.json

```diff
@@ -239,6 +239,11 @@
       "projectType": "library",
       "sourceRoot": "packages/react-conformance-make-styles/src"
     },
+    "@fluentui/react-conformance-griffel": {
```

Comment:
interesting how come that pipeline passed. this needs to be added to `nx.json` as well ( we are not on nx 13 yet  https://github.com/microsoft/fluentui/pull/21317 )

Reply:
No idea 😀 

P.S. I've pulled changes `master` and ran locally `yarn nx workspace-lint`, it's passing on Nx 13

Reply:
looks like nx has a regression ! 😶‍🌫️ thanks for double checking!

Reply:
https://github.com/nrwl/nx/issues/8666 - affects only nx 12 and below